CONTRIBUTING.md

```diff
@@ -0,0 +1,34 @@
+# Contributing
+
+## Developer guide
+
+ A [complete developer guide](https://docs.qgis.org/latest/en/docs/developers_guide/index.html) is also available to explain the build process, the unit testing system and more!
```

Comment:
The [install.md](https://github.com/qgis/QGIS/blob/master/INSTALL.md) file provides good references for build process and is worth a mention IMHO.

Reply:
I changes the 1st section, does it suite you?